src/torchpme/_utils.py

```diff
@@ -0,0 +1,108 @@
+from typing import Optional
+
+import torch
+
+
+def _validate_parameters(
+    charges: torch.Tensor,
+    cell: torch.Tensor,
+    positions: torch.Tensor,
+    neighbor_indices: torch.Tensor,
+    neighbor_distances: torch.Tensor,
+    smearing: Optional[float],
```

Comment:
```suggestion
    smearing: Optional[float] = None,
```
I guess you missed ` = None` here?

Reply:
Good catch. I would rather keep it as required argument and change it to `Union[float, None]`.